example/burrow_integration/permissions.py

```diff
@@ -66,7 +66,7 @@ def grant_permission(adress, permission):
     return hex_hash
 
 @integration_helpers.trace
-def revoke_permission(adress, permission):
+def revoke_permission(address, permission):
```

Comment:
remember also about changing the name here:
` "CallEngine", caller=ADMIN_ACCOUNT_ID, callee=address, input=params` (line 83)

Reply:
It was already named `address`.

Reply:
You are right. I don't know why I didn't notice:D.